### PR TITLE
update providers.xml to focus down the list of domains for Mailo

### PR DESCRIPTION
**Pull request by abawany**
# Important

Please confirm that you:

* read the [contributing section](https://github.com/M66B/FairEmail#user-content-contributing) (confirmed)
* agree to [the license and the copyright](https://github.com/M66B/FairEmail#user-content-license) (confirmed)

Thanks for your intention to contribute to the project!

### CODE REVIEW

File: app/src/main/res/xml/providers.xml [hunk 1]
```diff
@@ -1301,7 +1301,7 @@
     </provider>
     <provider
         name="Mailo.com"
-        domain="mailo\..*,net-c\..*,netc\..*,netcourrier.com,netcmail.com"
+        domain="africamel\\.net,americamel\\.net,asiamel\\.net,brusseler\\.com,calidifontain\\.be,email\\.bzh,emailasso\\.net,europamel\\.net,francemel\\.fr,inmano\\.com,lavache\\.com,mailo\..*,mailoo\\.org,monemail\\.com,mongenie\\.com,net-c\\.be,net-c\\.ca,net-c\\.cat,net-c\\.com,net-c\\.es,net-c\\.fr,net-c\\.it,net-c\\.lu,net-c\\.nl,net-c\\.pl,netc\\.eu,netc\\.fr,netc\\.it,netc\\.lu,netc\\.pl,netcmail\\.com,netcourrier\\.com,perso\\.be,postel\\.bzh,teaser\\.fr"
```
Comment (M66B) [inline app/src/main/res/xml/providers.xml:1304]: mailo\..* should be mailo\\..*

Comment (abawany) [reply app/src/main/res/xml/providers.xml:1304]: But it is ? Here is the excerpt from the diff: `...lavache\\.com,mailo\..*,mailoo\\.org,...`

Comment (abawany) [reply app/src/main/res/xml/providers.xml:1304]: I think I understand: it should be `mailo\\..*` instead of `mailo\..*` .

Comment (M66B) [reply app/src/main/res/xml/providers.xml:1304]: Yes, and GitHub 'swallowed' the slashes in my remark, which I just fixed.

Comment (abawany) [reply app/src/main/res/xml/providers.xml:1304]: Thank you. I see the change is merged in the current master as well so I appreciate that.

Comment (M66B) [reply app/src/main/res/xml/providers.xml:1304]: Yes, to save use both work I fixed the "mail.*" entry :-)

Comment (M66B) [reply app/src/main/res/xml/providers.xml:1304]: Thanks for your contribution!

**Pull request closed without merging**